rfcs/text/0014-extend-file-pe.md

```diff
@@ -85,9 +86,10 @@ Stage 3: Add more real world example source documents so we have at least 2 tota
 
 ## Scope of impact
 
-There should be no breaking changes, depreciation strategies, or significant refactoring as this is extending the existing fieldset.
+There should be no breaking changes, depreciation strategies, or significant refactoring as this is creating a sub-field for the existing file. fieldset.
```

Comment:
For consistency of language, consider replacing "the existing file. fieldset" with "the `file` top-level fieldset"